src/mixed-line-bar-chart/interfaces.ts

```diff
@@ -6,9 +6,7 @@ import { CartesianChartProps } from '../internal/components/cartesian-chart/inte
 
 export type ChartDataTypes = number | string | Date;
 
-export type SeriesType = 'line' | 'bar';
```

Comment:
Never used. The actually used type for this use case is declared at `IDataSeries['type']`